README.md

```diff
@@ -1,39 +1,48 @@
+* Migrating **Angular** + Rails to React + Rails. You can see an example of React on Rails and our work converting Angular to React on Rails at [egghead.io](https://egghead.io/browse/frameworks).
+* Augmenting your team to get your product completed more efficiently and quickly.
 
-**Speaking!**: Justin is speaking [GORUCO](http://goruco.com/#speakers) on Saturday, June 24, 2017 in NYC: "Front-End Sadness to Happiness: The React on Rails Story". [Email Justin](mailto:justin@shakacode.com) to give this talk at your local meetup or company. I'll also be discussing Webpacker versus or with React on Rails! [Subscribe](https://app.mailerlite.com/webforms/landing/l1d9x5) to be notified of local and *online* presentations of this talk. See the dates of other talks [here](https://forum.shakacode.com/t/talks-on-react-on-rails-june-2017/891), including: Tuesday, June 13th at Pivotal Labs, Santa Monica, CA. Tuesday, June 20th at Pivotal Labs, San Francisco, CA. Thursday, June 22nd in NYC. Tuesday, June 27th at Austin on Rails. Details [here](https://forum.shakacode.com/t/talks-on-react-on-rails-june-2017/891).
+My article "[Why Hire ShakaCode?](https://blog.shakacode.com/can-shakacode-help-you-4a5b1e5a8a63#.jex6tg9w9)" provides additional details about our projects.
 
-**Aloha from Justin Gordon** ([bio](http://www.railsonmaui.com/about)) and the [ShakaCode](http://www.shakacode.com) Team! We're actively looking for new projects involving React, React-Native, and Rails, including conversion of AngularJs to React. Please [contact me](mailto:justin@shakacode.com) if we could potentially help you in any way. Besides consulting on bigger projects, [ShakaCode](http://www.shakacode.com) is doing ScreenHero plus Slack/Github based [coaching](http://www.shakacode.com/work/shakacode-coaching-plan.pdf) for React on Rails. See our blog post [Can ShakaCode Help You?](https://blog.shakacode.com/can-shakacode-help-you-4a5b1e5a8a63#.jex6tg9w9) for more information.
+If any of this resonates with you, please email me, [justin@shakacode.com](mailto:justin@shakacode.com). I offer a free half-hour project consultation, on anything from React on Rails to any aspect of web or mobile application development for both consumer and enterprise products. 
 
-I'm offering a free half-hour project consultation, on anything from React on Rails to any aspect of web application development for both consumer and enterprise products. In addition to React.js and Rails, we're doing React-Native iOS and Android apps!
+We are **[currently looking to hire](http://www.shakacode.com/about/#work-with-us)** like-minded developers that wish to work on our product, [Friends and Guests](https://www.friendsandguests.com). By working on our product, new hires get trained in the ShakaCode way.
```

Comment:
I don't think that it is clear which project new hires would be working on.  I would simply say that Shaka Code is hiring and give them contact info here.